syntax = "proto3";

service Missile_Approaching{
    rpc missile_approaching(missile) returns (missile_status){}
}
message missile
{
    int64 x_pos = 1;
    int64 y_pos = 2;
    int64 hit_time = 3;
    string missile_type = 4;
}

message missile_status
{
    
}

